Source/Core/DolphinQt/Achievements/AchievementsWindow.cpp

```diff
@@ -28,11 +28,12 @@ AchievementsWindow::AchievementsWindow(QWidget* parent) : QDialog(parent)
   CreateMainLayout();
   ConnectWidgets();
   AchievementManager::GetInstance().SetUpdateCallback(
-      [this] { QueueOnObject(this, &AchievementsWindow::UpdateData); });
+      [this](AchievementManager::UpdatedItems updated_items) {
+        QueueOnObject(this,
+                      [this, updated_items] { AchievementsWindow::UpdateData(updated_items); });
```

Comment:
`[this, updated_items = std::move(updated_items)] { AchievementsWindow::UpdateData(std::move(updated_items)); });`